### PR TITLE
[feat|fix] : SDE frontend : Hotfix 3.2 with pcf - rel:2.3.0

### PR DESCRIPTION
**Pull request by sachinargade123**
### Fixed

- Code optimisation for download file from server, Generic error handling, UI improvement .
- trivy workflow updated.
- version bump for chart release.
- PCF documentation update

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by ajithsimons** [comment]
@almadigabor  Could you please help us create dependency review tickets for the below libraries?
npm/npmjs/@babel/code-frame/7.23.4
npm/npmjs/@babel/helper-string-parser/7.23.4
npm/npmjs/@babel/highlight/7.23.4

Thanks!

**Comment by carslen** [comment]
Done:

```
[main] INFO Querying Eclipse Foundation for license data for 500 items.
[main] INFO Found 145 items.
[main] INFO Querying Eclipse Foundation for license data for 500 items.
[main] INFO Found 94 items.
[main] INFO Querying Eclipse Foundation for license data for 290 items.
[main] INFO Found 51 items.
[main] INFO Querying ClearlyDefined for license data for 500 items.
[main] INFO Found 500 items.
[main] INFO Querying ClearlyDefined for license data for 500 items.
[main] INFO Found 500 items.
[main] INFO License information could not be automatically verified for the following content:
[main] INFO
[main] INFO npm/npmjs/@babel/code-frame/7.23.4
[main] INFO npm/npmjs/@babel/helper-string-parser/7.23.4
[main] INFO npm/npmjs/@babel/highlight/7.23.4
[main] INFO
[main] INFO This content is either not correctly mapped by the system, or requires review.
[main] INFO A review is required for npm/npmjs/@babel/highlight/7.23.4.
[main] INFO A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11658 .
[main] INFO A review is required for npm/npmjs/@babel/code-frame/7.23.4.
[main] INFO A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11659 .
[main] INFO A review is required for npm/npmjs/@babel/helper-string-parser/7.23.4.
[main] INFO A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11660 .
```

Tickets created:
- https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11658
- https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11659
- https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11660

File: DEPENDENCIES [hunk 1]
```diff
@@ -976,22 +964,21 @@ npm/npmjs/-/yocto-queue/0.1.0, MIT, approved, clearlydefined
 npm/npmjs/@adobe/css-tools/4.0.1, MIT, approved, #6345
 npm/npmjs/@ampproject/remapping/2.2.0, Apache-2.0, approved, clearlydefined
 npm/npmjs/@apideck/better-ajv-errors/0.3.6, MIT, approved, clearlydefined
-npm/npmjs/@babel/code-frame/7.12.11, MIT, approved, clearlydefined
-npm/npmjs/@babel/code-frame/7.18.6, MIT, approved, clearlydefined
+npm/npmjs/@babel/code-frame/7.23.4, , restricted, clearlydefined
```
Comment (carslen) [inline DEPENDENCIES:967]: Please update `DEPENDENCIES` file after successfull IP-Check for restricted 3rd party libs.

**Review by carslen — CHANGES_REQUESTED** [review]

**Comment by ajithsimons** [comment]
@carslen  thanks for the quick support! Dependencies file has been updated. Please verify!

**Comment by MehranRoshandel** [comment]
@carslen, can you please help us to merge this PR by today?

**Comment by adityagajbhiye9** [comment]
Updated.
#41  Legal documents updated for docker image.

**Comment by adityagajbhiye9** [comment]
fixed
#41 Updated Docker file for legal information

**Comment by hzierer** [comment]
@carslen Dockerfile LGTM now

**Comment by FaGru3n** [comment]
Hi all,

in my opinion this is really hard to verfiy and IMHO i would reject this PR.

This looks for me to one PR to fix all problems...  👎🏼 not a good practice in my perspective.

Reason: 
- Hard to review
- No transparency on what change fixed which issue  without dedicated commit messages, etc.
- Hard/impossible to create a proper changelog

The Best practices we nomally use 1 problem like docs / 1 pr / dedicated commit messages why to change and what to improve.

if i read the description correctly that would be at least 6 PR´s like:

1. Code optimisation for download file from server
2. Generic error handling
3. UI improvement
4. trivy workflow updated
5. version bump for chart release
6. PCF documentation update

Only guessing is it possible for you @sachinargade123 to split all your changes in separate PR´s or generating new ones? This is also the reason why we as a system team generate multible issues for each TRG seaction.

**Comment by sachinargade123** [comment]
We are closing this PR. We will create new PR's.

**Pull request closed without merging**